openedx/adg/admin_dashboard/constants.py

```diff
@@ -2,4 +2,9 @@
 Constants for `admin_dashboard`
 """
 DEFAULT_PAGINATE_BY = 20
-PAGINATE_BY_OPTIONS = (10, 20, 50, 100)
+PAGINATE_BY_OPTIONS = [
+    (10, 'Show 10'),
+    (20, 'Show 20'),
+    (50, 'Show 50'),
+    (100, 'Show 100')
+]
```

Comment:
Are we displaying them on pages? Should these be translated?

Reply:
Good catch, they should be. Fixing!